syntax="proto3";

package proto_students;

message FullName{
    string first_name = 1;
    string second_name = 2;
    optional string patronymic = 3;
}

message Student{
    FullName name = 1;
    repeated double grade = 2;
    double average_grade = 3;
}

message StudentsGroup{
    repeated Student students = 1;
}
